corehq/apps/reports/standard/export.py

```diff
@@ -280,6 +280,7 @@ def _sortkey(form):
                 if group.form_exports],
             report_slug=self.slug,
             property_hash=self.properties(size_hash),
+            has_media=self.request.project.has_media(),
```

Comment:
I don't think this is the right `has_media`. i think the project-level attribute refers to multimedia in the app, whereas the export should check whether there are any media submissions. @twymer did you test this?